stdlib/io/src/test/java/org/ballerinalang/stdlib/io/IOTest.java

```diff
@@ -270,28 +271,37 @@ public void appendCharacters() throws IOException {
     }
 
     @Test(description = "Test 'writeRecords' function in ballerina/io package")
-    public void testWriteRecords() {
-        String[] content = {"Name", "Email", "Telephone"};
+    public void testWriteRecords() throws URISyntaxException {
+        String[] content = { "Name", "Email", "Telephone" };
         BValueArray record = new BValueArray(content);
         String sourceToWrite = currentDirectoryPath + "/recordsFile.csv";
-
         //Will initialize the channel
-        BValue[] args = {new BString(sourceToWrite), new BString("UTF-8"), new BString("\n"), new
-                BString(",")};
+        BValue[] args = {
+                new BString(sourceToWrite), new BString("UTF-8"), new BString("\n"), new BString(",")
+        };
         BRunUtil.invoke(recordsInputOutputProgramFile, "initWritableChannel", args);
-
-        args = new BValue[]{record};
-        BRunUtil.invoke(recordsInputOutputProgramFile, "writeRecord", args);
-
+        BValue[] recordArgs = new BValue[] { record };
+        BRunUtil.invoke(recordsInputOutputProgramFile, "writeRecord", recordArgs);
         BRunUtil.invoke(recordsInputOutputProgramFile, "closeWritableChannel");
+
+        BRunUtil.invoke(recordsInputOutputProgramFile, "initReadableChannel", args);
+        BValue[] returns = BRunUtil.invoke(recordsInputOutputProgramFile, "nextRecord");
+        BValueArray records = (BValueArray) returns[0];
+        Assert.assertEquals(records.getStringArray()[0], content[0], "Content mismatch in the return array");
```

Comment:
Shall we use a constant for the repeated String?

Reply:
What is the gain we are getting that?

Reply:
We eliminate duplicates. If reluctant to use a constant we can use a local variable. By this if one wants to change the content of this sentence in the future he/she needs to do it in only one place and need not repeat ourselves. Hope it's clear.

Reply:
This text used here to give some information regarding the assert failure. I don't think we need to have constant here in any sort of. It will reduce the code readability. In term of changing the text,  we are no longer using Vi kind editors :). Almost all the high level IDEs support replace these kind of text easily. Addressd the rewording but not going to make it constant.

Reply:
It is one of  sonar code smells: https://rules.sonarsource.com/java/RSPEC-1192

Reply:
In this sonar example, it clearly explains why we should avoid duplicates. In there, that word interconnected with each other. But here it's not.